kubernetes/kserve/kserve_wrapper/TorchserveModel.py

```diff
@@ -1,22 +1,35 @@
 """ The torchserve side inference end-points request are handled to
     return a KServe side response """
-import json
 import logging
 import pathlib
-from typing import Dict
+from enum import Enum
+from typing import Dict, Union
 
+import grpc
+import inference_pb2_grpc
 import kserve
-import tornado.web
+from gprc_utils import from_ts_grpc, to_ts_grpc
+from inference_pb2 import PredictionResponse
+from kserve.errors import ModelMissingError
 from kserve.model import Model as Model
-from kserve.model import ModelMissingError
+from kserve.protocol.grpc.grpc_predict_v2_pb2 import (
+    ModelInferRequest,
+    ModelInferResponse,
+)
+from kserve.protocol.infer_type import InferRequest, InferResponse
 
 logging.basicConfig(level=kserve.constants.KSERVE_LOGLEVEL)
 
+PREDICTOR_URL_FORMAT = PREDICTOR_V2_URL_FORMAT = "http://{0}/predictions/{1}"
+EXPLAINER_URL_FORMAT = EXPLAINER_v2_URL_FORMAT = "http://{0}/explanations/{1}"
```

Comment:
Do we need support backward compatible (ie. v1)?

Reply:
yes, there is no plan for deprecating v1 format in near future.